src/pet.js

```diff
@@ -0,0 +1,89 @@
+const MAX_FITNESS = 10
+const MIN_HUNGER = 0
+
```

Comment:
This clear line could be removed here so all your constants are grouped together